cmd/res/devices/general.usb.camera.toml.example

```diff
@@ -0,0 +1,9 @@
+[[DeviceList]]
+  Name = "example-camera"
+  ProfileName = "USB-Camera-General"
+  Description = "Example Camera"
+  Labels = [ "device-usb-camera-example" ]
+  [DeviceList.Protocols]
+    [DeviceList.Protocols.USB]
+    Path = "/dev/video0"
+    AutoStreaming = "false"
```

Comment:
This PR is adding that example.
A clean merge of main into branch does not affect the diff, because the diff is calculated between the branch and main. On the other hand, rebasing will require a re-review of everything because there is no guaranry that the author hasn't changed something else.

Reply:
Created a separate PR for the example. Will remove from this PR once merged:
- https://github.com/edgexfoundry/device-usb-camera/pull/33

Reply:
PR has been merged. Removed the example from this PR and merged main into the branch.